### PR TITLE
feat: display address segments, closes #2690

### LINKED ISSUE #2690: [Bitcoin] Address displayer

### PR DESCRIPTION
**Pull request by kyranjamie**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/3189062568).

image

New component that displays addresses in easily readable segments.

### CODE REVIEW

File: src/app/components/address-displayer/address-displayer.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  return [...text].reduce(textGrouper, initialValue).result;
+}
+
+interface AddressDisplayerProps {
+  address: string;
+}
+export function AddressDisplayer({ address }: AddressDisplayerProps) {
+  return (
```
Comment (fbwoolf) [inline src/app/components/address-displayer/address-displayer.tsx:25]:
```suggestion
  return groupByFour(address).map((letterGroup, index) => (
    <AddressDisplayerLayout isEven={isEven(index + 1)}>
      {letterGroup.join('')}
    </AddressDisplayerLayout>
  ));
```

File: src/app/pages/receive-tokens/receive-tokens.tsx [hunk 2]
```diff
@@ -1,32 +1,32 @@
 import { useNavigate } from 'react-router-dom';
 import toast from 'react-hot-toast';
 import { FiCopy } from 'react-icons/fi';
-import { Box, Flex, useClipboard, Stack, color, Button } from '@stacks/ui';
-import { truncateMiddle } from '@stacks/ui-utils';
-
-import { useWallet } from '@app/common/hooks/use-wallet';
+import { Box, Flex, useClipboard, color, Button } from '@stacks/ui';
```
Comment (fbwoolf) [inline src/app/pages/receive-tokens/receive-tokens.tsx:4]: I always try to separate library imports at top > then a section of alias imports > then local imports w/out an alias. I try to keep them separated in sections.

**Review by fbwoolf — APPROVED** [review]

File: src/app/components/address-displayer/address-displayer.tsx (hunk 1 above)
Comment (kyranjamie) [reply src/app/components/address-displayer/address-displayer.tsx:25]: I would love to be able to do this, but it gives typing errors

File: src/app/pages/receive-tokens/receive-tokens.tsx (hunk 2 above)
Comment (kyranjamie) [reply src/app/pages/receive-tokens/receive-tokens.tsx:4]: Not sure where it violates this rule? `@stacks/ui` is a library import then there's a line return, then  local aliased imports

File: src/app/components/address-displayer/address-displayer.tsx (hunk 1 above)
Comment (kyranjamie) [reply src/app/components/address-displayer/address-displayer.tsx:25]: image

File: src/app/pages/receive-tokens/receive-tokens.tsx (hunk 2 above)
Comment (kyranjamie) [reply src/app/pages/receive-tokens/receive-tokens.tsx:4]: Happy to change separately, but not sure where it's wrong

**Pull request merged**